Sources/FontInspector.swift

```diff
@@ -170,6 +170,14 @@
 
     }
 
+    extension Fractions: MappableFeature {
```

Comment:
#req move after `NumberSpacing` for consistency. I think the code may be inconsistent about whether number case or number spacing is first everywhere, but I'd at least like them to stick together.